src/game/ticTacToeGame.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+exports.__esModule = true;
+exports.TicTacToe = void 0;
+var events_1 = require("../events/events");
+var types_1 = require("../events/types");
+var gameTypes_1 = require("../shared/types/gameTypes");
+var playerTypes_1 = require("../shared/types/playerTypes");
+var TicTacToe = /** @class */ (function () {
+    function TicTacToe(toMove, gameType) {
+        if (toMove === void 0) { toMove = playerTypes_1.PlayerTypes.player1; }
```

Comment:
No need for IIFE functions when we have the 'Class' keyword.

`export class TicTacToe {...`